docker-compose.yml

```diff
@@ -9,3 +9,4 @@ services:
       POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
     volumes:
       - ./volumes/db:/var/lib/postgresql/data
+      - ./files/postgres/postgresql.conf:/etc/postgresql/postgresql.conf
```

Comment:
on the docker command line in linux, relative paths do not work, so `$PWD/` needs to be used instead of `./` - not sure if it is the same for docker-compose

Reply:
I'm cargo-culting from the line above. It seems that `./` isn't necessary per the docs. Regardless, this current syntax does seem to work on Ubuntu.

Reply:
phew, so likely just a `-v` on docker itself.